packages/react/src/create-strict-api/__tests__/__fixtures__/strict-api.ts

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
 import { createStrictAPI } from '@compiled/react';
```

Comment:
```
import { createStrictAPI } from '../../index';
```
instead of disabling linting rule.

Reply:
Oh no, I didn't see this! Would I be able to add this change to he Version Bump PR?